pkgs/development/libraries/sfml/sfml_git.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, cmake, mesa, libX11, freetype, libjpeg, openal, libsndfile
+, glew, libXrandr, libXrender
+}:
+stdenv.mkDerivation rec {
+  name = "sfml-git-20110428";
```

Comment:
Hm. I was thinking about this pkg and maybe we should drop git version since there is a stable one?